apis/cluster/composition.yaml

```diff
@@ -104,3 +104,22 @@ spec:
             kind: XServices
             resourceSelector:
               matchControllerRef: true
+
+    - name: usageXEksByArbitraryLabeledRelease
+      base:
+        apiVersion: apiextensions.crossplane.io/v1alpha1
+        kind: Usage
+        spec:
+          of:
+            apiVersion: aws.platform.upbound.io/v1alpha1
+            kind: XEKS
+            resourceSelector:
+              matchControllerRef: true
+          by:
+            apiVersion: helm.crossplane.io/v1beta1
+            kind: Release
+            resourceSelector:
+              matchLabels:
+                platform.upbound.io/deletion-ordering: enabled
+      readinessChecks:
```

Comment:
why need this ?

Reply:
@haarchri to make it relaxed - there can be a case when App / no other Releases with this special label are instantiated and we don't want to invalidated the Readiness of the Cluster in this case